service/seed-sampling-example-config.yaml

```diff
@@ -15,7 +15,7 @@ preemptible_runners: true
 
 # This experiment should generate a report that is combined with other public
 # "production" experiments.
-merge_with_nonprivate: true
+merge_with_nonprivate: false
```

Comment:
I suspect that the experiment did not use the new sampling strategy: Its report shows the results of 20 fuzzer trials running on each benchmark for 23 hours, which 1) disagrees with the experiment config, and 2) should not complete in 16 hours.
The strange behavior may be caused by this line, which allows re-using previous results. As a result, the experiment merges previous 23-hour 20-trial results (without seed sampling)with the new result (with sampling).

I pushed this commit to fix it and re-run the experiment.